src/Microsoft.AspNetCore.Server.Testing/Deployers/SelfHostDeployer.cs

```diff
@@ -49,23 +49,17 @@ public override DeploymentResult Deploy()
 
         private CancellationToken StartSelfHost(Uri uri)
         {
-            var commandName = DeploymentParameters.Command;
-            if (string.IsNullOrEmpty(commandName))
-            {
-                commandName = "run";
-            }
-
-            var dnxPath = Path.Combine(TargetRuntimeBinPath, DnxCommandName);
-            var dnxArgs = $"-p \"{DeploymentParameters.ApplicationPath}\" {commandName} " +
-                          $"--server.urls {uri} " +
-                          $"--server {(DeploymentParameters.ServerType == ServerType.WebListener ? "Microsoft.AspNetCore.Server.WebListener" : "Microsoft.AspNetCore.Server.Kestrel")}";
+            var dotnetArgs = $"run"
+                + $" -p \"{DeploymentParameters.ApplicationPath}\" {DotnetArgumentSeparator}"
+                + $" --server.urls {uri} "
+                + $" --server {(DeploymentParameters.ServerType == ServerType.WebListener ? "Microsoft.AspNetCore.Server.WebListener" : "Microsoft.AspNetCore.Server.Kestrel")}";
```

Comment:
so the parens is the the trick to make ternary work inside string interpolation... good to know - I fought it once...